EFCore.NamingConventions/Internal/LowerCaseFirstCharacterNameRewriter.cs

```diff
@@ -0,0 +1,14 @@
+using System.Globalization;
+
+namespace EFCore.NamingConventions.Internal
+{
+    public class LowerCaseFirstCharacterNameRewriter : INameRewriter
```

Comment:
Also need to rename to CamelCaseNameRewriter

Reply:
done, alsi renamed some others remaining, (see in last commit)